### PR TITLE
fix(events getting deselcted)

### PR DESCRIPTION
**Pull request by Pavan-Nambi**
related to  #223

image:
Screenshot_20231123_190655

### CODE REVIEW

**Comment by Pavan-Nambi** [comment]
will update page jump issue after getting reply for [this comment](https://github.com/CivicDataLab/civicdatalab.github.io/issues/223#issuecomment-1824436749)

**Comment by Pavan-Nambi** [comment]
Screenshot_20231125_084914

**Comment by Pavan-Nambi** [comment]
@PixeledCode

File: src/pages/events.js [hunk 1]
```diff
@@ -47,6 +47,9 @@ export const EventsContent = styled.div`
   display: grid;
   grid-template-rows: auto auto;
   grid-template-columns: 1fr;
+  min-height: 90vh;
+  align-items: start;
+  align-content: start;
```
Comment (Pavan-Nambi) [inline src/pages/events.js:52]: have used `align-content` and `align-items` here coz event contents are taking extra space it gets from height and centering/spacing themselves(idk better way to explain this verbally lol )

**Review by PixeledCode — APPROVED** [review]

**Pull request merged**